tests/io/datasets/test_seq.py

```diff
@@ -120,12 +120,86 @@ def test_positive_sync_offset(default_seq, lt_ctx):
+                 'Rows="2048" Columns="2048"><Defect Rows="1155-1156"/><Defect Rows="1706-1707"/></BadPixelMap></BadPixels>' \
+                 '</Configuration>'
+    metadata = {
+        "UnbinnedFrameSizeX": 1024,
+        "UnbinnedFrameSizeY": 1024,
+        "OffsetX": 1536,
+        "OffsetY": 1536,
+        "HardwareBinning": 1
+    }
+    test_arr = np.zeros((1024, 1024), dtype=bool)
+    test_arr[775] = True
+    test_arr[:, 775] = True
+    test_arr[776] = True
+    expected_res = _load_xml_from_string(xml=xml_string, metadata=metadata)
+    assert np.array_equal(expected_res.todense(), test_arr)
+
+
+def test_xml_excluded_pixels_only_binned():
+    xml_string = '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>' \
+                 '<Configuration><PixelSize></PixelSize><DiffPixelSize></DiffPixelSize><BadPixels><BadPixelMap ' \
+                 'Rows="4096" ' \
+                 'Columns="4096"><Defect Rows="2311-2312"/><Defect Rows="3413-3414"/><Defect Columns="1310-1312"/>' \
+                 '<Defect Column="1300"/><Defect Row="100" Column="150" /></BadPixelMap><BadPixelMap Binning="2" ' \
+                 'Rows="2048" Columns="2048"><Defect Rows="1155-1156"/><Defect ' \
+                 'Rows="1706-1707"/></BadPixelMap></BadPixels>' \
+                 '</Configuration>'
+    metadata = {
+        "UnbinnedFrameSizeX": 4096,
+        "UnbinnedFrameSizeY": 4096,
+        "OffsetX": 0,
+        "OffsetY": 0,
+        "HardwareBinning": 4
+    }
+    test_arr = np.zeros((1024, 1024), dtype=bool)
+    test_arr[577] = True
+    test_arr[578] = True
+    test_arr[853] = True
+    test_arr[25, 37] = True
+    test_arr[:, 325] = True
+    test_arr[:, 327:329] = True
+    expected_res = _load_xml_from_string(xml=xml_string, metadata=metadata)
+    assert np.array_equal(expected_res.todense(), test_arr)
+
+
+def test_xml_excluded_pixels_cropped_binned():
+    xml_string = '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>' \
+                 '<Configuration><PixelSize></PixelSize><DiffPixelSize></DiffPixelSize><BadPixels><BadPixelMap ' \
+                 'Rows="4096" ' \
+                 'Columns="4096"><Defect Columns="1310-1312"/><Defect Column="1300"/><Defect ' \
+                 'Rows="2311-2312"/><Defect ' \
+                 'Rows="3413-3414"/></BadPixelMap><BadPixelMap Binning="2" ' \
+                 'Rows="2048" Columns="2048"><Defect Rows="1155-1156"/><Defect ' \
+                 'Rows="1706-1707"/></BadPixelMap></BadPixels>' \
+                 '</Configuration>'
+    metadata = {
+        "UnbinnedFrameSizeX": 2048,
+        "UnbinnedFrameSizeY": 2048,
+        "OffsetX": 1024,
+        "OffsetY": 1024,
+        "HardwareBinning": 2
+    }
+    test_arr = np.zeros((1024, 1024), dtype=bool)
+    test_arr[644] = True
+    test_arr[643] = True
+    test_arr[:, 143:145] = True
+    test_arr[:, 138] = True
```

Comment:
Since it uses the 4096*4096 map we will we first crop it to the size of 2048 in the middle part(because of the offset) according to the metadata that means we substract 1024 from the column coordinates of the bad pixel map which will be 286 287 288 276 if we bin these values we will get 143 143 144 138